src/Contrib/OtlpHttp/Exporter.php

```diff
@@ -8,40 +8,27 @@
 use Http\Discovery\Psr18ClientDiscovery;
 use InvalidArgumentException;
 use OpenTelemetry\API\Common\Signal\Signals;
-use OpenTelemetry\Contrib\Otlp\SpanConverter;
-use Opentelemetry\Proto\Collector\Trace\V1\ExportTraceServiceResponse;
-use OpenTelemetry\SDK\Behavior\LogsMessagesTrait;
+use OpenTelemetry\Contrib\Otlp\SpanExporter;
 use OpenTelemetry\SDK\Common\Environment\EnvironmentVariablesTrait;
 use OpenTelemetry\SDK\Common\Environment\Variables as Env;
 use OpenTelemetry\SDK\Common\Export\Http\PsrTransportFactory;
 use OpenTelemetry\SDK\Common\Export\TransportFactoryInterface;
 use OpenTelemetry\SDK\Common\Export\TransportInterface;
-use OpenTelemetry\SDK\Common\Future\CancellationInterface;
-use OpenTelemetry\SDK\Common\Future\FutureInterface;
 use OpenTelemetry\SDK\Common\Otlp\HttpEndpointResolver;
 use OpenTelemetry\SDK\Trace\SpanExporterInterface;
 use function sprintf;
-use Throwable;
 
-class Exporter implements SpanExporterInterface
+class Exporter
```

Comment:
`OtlpHttp` and `OtlpGrpc` should be removed, kept for BC for now.

Reply:
how about a `@deprecated` class annotation (or, even raising a deprecation notice in the constructor) to warn users?

Reply:
We don't provide an alternative yet - will be deprecated/replaced in #838.